scripts/ChangesSinceLastRelease.txt

```diff
@@ -172,3 +172,52 @@ Windows.Win32.UI.WindowsAndMessaging.Apis.SetLayeredWindowAttributes : crKey...U
 Windows.Win32.UI.WindowsAndMessaging.Apis.SetSysColors : lpaRgbValues...UInt32* => COLORREF*
 Windows.Win32.UI.WindowsAndMessaging.Apis.UpdateLayeredWindow : crKey...UInt32 => COLORREF
 Windows.Win32.UI.WindowsAndMessaging.UPDATELAYEREDWINDOWINFO.crKey...System.UInt32 => Windows.Win32.Foundation.COLORREF
+# LOGFONT enum changes
+Windows.Win32.Graphics.Gdi.Apis.ANSI_CHARSET removed
+Windows.Win32.Graphics.Gdi.Apis.ARABIC_CHARSET removed
+Windows.Win32.Graphics.Gdi.Apis.BALTIC_CHARSET removed
+Windows.Win32.Graphics.Gdi.Apis.CHINESEBIG5_CHARSET removed
+Windows.Win32.Graphics.Gdi.Apis.DEFAULT_CHARSET removed
+Windows.Win32.Graphics.Gdi.Apis.EASTEUROPE_CHARSET removed
+Windows.Win32.Graphics.Gdi.Apis.FW_BLACK removed
+Windows.Win32.Graphics.Gdi.Apis.FW_BOLD removed
+Windows.Win32.Graphics.Gdi.Apis.FW_DEMIBOLD removed
+Windows.Win32.Graphics.Gdi.Apis.FW_DONTCARE removed
+Windows.Win32.Graphics.Gdi.Apis.FW_EXTRABOLD removed
+Windows.Win32.Graphics.Gdi.Apis.FW_EXTRALIGHT removed
+Windows.Win32.Graphics.Gdi.Apis.FW_HEAVY removed
+Windows.Win32.Graphics.Gdi.Apis.FW_LIGHT removed
+Windows.Win32.Graphics.Gdi.Apis.FW_MEDIUM removed
+Windows.Win32.Graphics.Gdi.Apis.FW_NORMAL removed
+Windows.Win32.Graphics.Gdi.Apis.FW_REGULAR removed
+Windows.Win32.Graphics.Gdi.Apis.FW_SEMIBOLD removed
+Windows.Win32.Graphics.Gdi.Apis.FW_THIN removed
+Windows.Win32.Graphics.Gdi.Apis.FW_ULTRABOLD removed
+Windows.Win32.Graphics.Gdi.Apis.FW_ULTRALIGHT removed
```

Comment:
This diff output doesn't indicate if all these members survived and were added to Windows.Win32.Graphics.Gdi.FONT_WEIGHT. So fingers crossed! (Not a blocker)

Reply:
I will load it up in ilspy and confirm. I don't want to break anything.

Reply:
Verified.